docs/containers/debug-python.md

```diff
@@ -0,0 +1,149 @@
+---
+Area: containers
+ContentId: f9ffec31-9253-4f71-a4eb-79ea7b3a8f55
+PageTitle: Configure and troubleshoot debugging of Python apps running in a Docker container
+DateApproved: 01/29/2020
+MetaDescription: How to configure and troubleshoot debugging of Python apps running in a Docker container, using Visual Studio Code.
+---
+
+# Debug Python within a container
+
+When adding Docker files to a Python project, tasks and launch configurations are added to enable debugging the application within a Docker container. To accommodate the various scenarios of Python projects, some apps may require additional configuration.
+
+## Configuring the Docker container entry point
+
+The Docker extension infers the entry point of the Docker container via properties of `tasks.json`. This container entry point is automatically configured when using the **Docker: Add Docker Files to Workspace...** command.
+
+### Example: Configuring the entry point for a Python module
+
+```json
+{
+  "tasks": [
+    {
+      "type": "docker-run",
+      "label": "docker-run: debug",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "python": {
+        "args": [
+          "runserver",
+          "0.0.0.0:8000",
+          "--nothreading",
+          "--noreload"
+        ],
+        "module": "manage"
+      }
+    }
+  ]
+}
+```
+
+### Example: Configuring the entry point for a Python file
+
+```json
+{
+  "tasks": [
+    {
+      "type": "docker-run",
+      "label": "docker-run: debug",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "python": {
+        "file": "hello_app/webapp.py"
+      }
+    }
+  ]
+}
+```
+
+## Automatically launching the browser to the entry page of the application
+
+When the **Docker: Python - Django** or **Docker: Python - Flask** launch configurations are selected, the Docker extension will automatically launch the browser to the main page of the app. While this feature is enabled by default, this behavior can be configured explicitly via the `dockerServerReadyAction` object in `launch.json`.
```

Comment:
Can you rephrase to make it more applicable to the user?  Something like "You can select the **Docker: Python - Django** or **Docker: Python - Flask** launch configuration to automatically launch the browser..." and then later "you can configure this behavior explicitly by setting the `dockerServerReadyAction` object in *launch.json*.  BTW, I think filenames should be in italics (single asterisk) rather than code.

Reply:
Thanks, I made the updates to be a more applicable to the user. In terms of single asterisks on the filename, for consistency I will keep it as single ticks. The reason why is this is done throughout the containers section when referring to any file.